code/client/munkiimport

```diff
@@ -800,6 +819,10 @@ def main():
         configure()
         exit(0)
 
+    if options.init:
+        init_repo(arguments[0].strip())
```

Comment:
There's a difference between doing an rstrip() of raw_input during --configure and stripping spaces from the _already parsed arguments_ the shell has handed you. I think you should not do the latter at all.